docs/api/sql/Raster-operators.md

```diff
@@ -192,6 +192,14 @@ Provided band index 3 does not lie in the raster
 
 Introduction: Returns the georeference metadata of raster as a string in GDAL or ESRI format. Default is GDAL if not specified.
 
+!!!note
+    If you are using `show()` to display the output, it will show special characters as escape sequences. To get the expected behavior use the following code:
+
+    ```sql
+    print(df.selectExpr("RS_GeoReference(rast)").sample(0.5).collect().mkString(""))
```

Comment:
This is not a SQL example. This should be a language-specific example. You need to provide examples for (Scala/Java) and (Python). Please also say `sample()` here is to reduce data to be collected. Other functions such as filter() will work as well.

Reply:
I don't think the filter function makes much sense here. As there's nothing to filter, the function returns one row, that's a string with special characters for formatting.

Reply:
@furqaankhan sample and filter are operations that can reduce the number of resulting rows. Without this kind of commands, a direct collect() on a DataFrame that has millions of row will crash the driver program.